packages/datadog-plugin-cypress/test/index.spec.js

```diff
@@ -34,6 +34,9 @@ describe('Plugin', function () {
       return agent.load()
     })
     beforeEach(function (done) {
+      // This test does not check test optimization agentless protocol,
+      // so we'll make the tracer default to reporting to v0.4/traces
+      agent.setAvailableEndpoints([])
```

Comment:
ℹ️ Necessary because of the change in `packages/datadog-plugin-cypress/test/app/cypress/plugins/index.js`. Now the tracer is initialized in test visibility mode, as it should.